appIos/feed/feed/FeedScreen.swift

```diff
@@ -44,6 +46,10 @@ public struct FeedScreenContent: View {
                 )
             }
         }
+        .onChange(of: searchQuery) { _ in
+            viewModel.onSearchTextChanged(searchQuery: searchQuery)
+        }
+        .searchable(text: $searchQuery, prompt: "Search Feed")
```

Comment:
According to what I read, this should go at the top view and passing down the bindings.
We may need a `onSubmit`, but it seems like one does search by typing and not usually 'submitting' the search?

Reply:
Looking great - don't think `onSubmit` is necessary here for the reason you mentioned. My only suggestion is to update the prompt to something like `Search articles` as we'd effectively be searching for articles with the search term.

Reply:
Removed onSubmit and changed prompt

Reply:
@gchristov what would be the `placement` here? `.automatic` is the default but you may need to drag once to make it appear with some screens (some some examples about that...), would it be advisable to make it relative to the top or something?

Reply:
I think it’s expected to have to drag down first to show the search prompt - standard iOS behaviour, so I’d keep it like that for now 👍